jnigen/lib/src/logging/logging.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+enum LogLevel {
```

Comment:
We already have these in package logging. Why do we need a duplicate abstraction?

Reply:
I didn't know there was a standard logging package, and defined logging levels + logger. I wrote this class more similar to android's logging levels.

I should probably update this. I will update this PR tomorrow.

Re: unit test, I think it's more effort than worth, we have to update expected output every time we add a logging statement in code.

Reply:
Yeah, maybe leave the unit test out, unless you for example use the logger to error on the --config path not being found. That we wouldn't change that so often.